### DISCUSSION TITLE
WiX Online Meeting 270

### DISCUSSION THREAD

**Discussion by robmen** [wixtoolset/issues#7803] [Announcements]
The next WiX Online meeting will be on October 31st, 2023 @9:30 AM Pacific.

Agenda items:

* Triage

Join us live or later at: https://www.youtube.com/@wixtoolset/

**Comment by Federau** [comment]
Hi Robmen，Can you share the live address?

**Reply by robmen** [reply]
No, because the address for the meeting changes. You can go to the WiX YouTube page, and the live stream will be listed first.

**Comment by robmen** [comment]
Meeting highlights and video can be found at https://www.firegiant.com/blog/2023/10/31/wix-online-meeting-270-highlights/
